webapp/sources/ldap-inventory/inventory-provisioning-core/src/main/scala/com/normation/inventory/ldap/provisioning/SoftwareIdFinder.scala

```diff
@@ -90,7 +90,7 @@ class NameAndVersionIdFinder(
     } yield {
       //now merge back
       entities.foldLeft(MergedSoftware(Set(),Set())) { case(ms, s) =>
-        merged.find { x => x.name == s.name && x.version == s.version } match {
+        merged.find { x => x == s } match {
```

Comment:
I don't think we want to compare ALL fields ? Only name, version, source name, source version ?